src/app/+item-page/edit-item-page/item-relationships/edit-relationship/edit-relationship.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, Input, OnChanges } from '@angular/core';
+import { FieldUpdate } from '../../../../core/data/object-updates/object-updates.reducer';
+import { cloneDeep } from 'lodash';
+import { Item } from '../../../../core/shared/item.model';
+import { ObjectUpdatesService } from '../../../../core/data/object-updates/object-updates.service';
+import { FieldChangeType } from '../../../../core/data/object-updates/object-updates.actions';
+import { ItemViewMode } from '../../../../shared/items/item-type-decorator';
+
+@Component({
+  // tslint:disable-next-line:component-selector
+  selector: '[ds-edit-relationship]',
+  styleUrls: ['./edit-relationship.component.scss'],
+  templateUrl: './edit-relationship.component.html',
+})
+export class EditRelationshipComponent implements OnChanges {
+  /**
+   * The current field, value and state of the relationship
+   */
+  @Input() fieldUpdate: FieldUpdate;
+
+  /**
+   * The current url of this page
+   */
+  @Input() url: string;
+
+  /**
+   * The related item of this relationship
+   */
+  item: Item;
+
+  /**
+   * The view-mode we're currently on
+   */
+  viewMode = ItemViewMode.Element;
+
+  constructor(private objectUpdatesService: ObjectUpdatesService) {
+  }
+
+  /**
+   * Sets the current relationship based on the fieldUpdate input field
+   */
+  ngOnChanges(): void {
+    this.item = cloneDeep(this.fieldUpdate.field) as Item;
+  }
+
+  /**
+   * Sends a new remove update for this field to the object updates service
+   */
+  remove(): void {
+    this.objectUpdatesService.saveRemoveFieldUpdate(this.url, this.item);
```

Comment:
just after removing the relationship, the related item still appears in the "Items relationships" tab. And you can even try to repeat the same action (you will get an error - 
![2019-06-18 12_21_29-Item Edit - Relationships](https://user-images.githubusercontent.com/877488/59678272-df267300-91c3-11e9-9527-3f8433676ff0.png)
).
You need to switch to any other tab and then go back to Items relationships tab in order to be updated.

Reply:
Hi @paulo-graca
This was apparently caused by some updates in master from REST
Kristof has solved these errors now, and I was able to use delete relationships

Reply:
Ok, thank you @benbosman! I will perform some more tests.

Reply:
Thank you @benbosman and @Atmire-Kristof , that fix that issue.
But just after removing that relationship, that item is still showed as a part of Works associated with that Person.
![2019-07-01 17_00_40-dev5 rcaap pt_items_3f685bbd-07d9-403e-9de2-b8f0fabe27a7](https://user-images.githubusercontent.com/877488/60450695-4d3a5380-9c22-11e9-9ddd-2a820c64db94.png)

and you click on that entry, you will see that the item is no longer related:
![2019-07-01 17_00_56-dev5 rcaap pt_items_8e4314ca-2f60-4b0d-aa1f-48acb9aee6ed](https://user-images.githubusercontent.com/877488/60450733-60e5ba00-9c22-11e9-8fed-191398468f6d.png)

It seems to me that is a caching issue.

Reply:
Thanks @paulo-graca
I've asked Kristof to verify it in detail, it seems that no new REST discovery is created for displaying the item page. The discovery cache would need to be invalidated here.

Do you know how this is handled for other situations where the discovery cache has to be invalidated (e.g. a new item is submitted, item metadata has been edited, …)?

Reply:
It was indeed a caching issue. This should be fixed now.

Reply:
Ok, thank you @Atmire-Kristof and @benbosman! I will perform another test.